internal/msalbase/authentication_result_test.go

```diff
@@ -7,85 +7,91 @@ import (
 	"reflect"
 	"testing"
 	"time"
+
+	"github.com/kylelemons/godebug/pretty"
```

Comment:
>"github.com/kylelemons/godebug/pretty" [](start = 1, length = 38)

any reason for this? if so why not everywhere?

Reply:
pretty can display differences in two types in a human readable way that reflect.DeepEqual() doesn't give and the other package you had(something I think was called go-render, which I removed) doesn't do well.  Kyle was a Go author, this is used all the time inside Google, so its a package I trust, except when dealing with protocol buffers (you have to use special compare package for those).  

It hasn't gone everywhere because I haven't fixed up every test.  As I have to touch tests, I replace as much as I can to make the output actually readable for errors.  

In the end, there will be no reflect.DeepEqual() and for comparisons of structs, I will always use pretty.